lib/pages/search/search_text_field.dart

```diff
@@ -39,11 +39,18 @@ class SearchTextField extends StatelessWidget {
             size: SearchViewStyle.searchIconSize,
             color: Theme.of(context).colorScheme.onSurface,
           ),
-          suffixIcon: TwakeIconButton(
-            tooltip: L10n.of(context)!.close,
-            icon: Icons.close,
-            onTap: () {
-              textEditingController.clear();
+          suffixIcon: ValueListenableBuilder(
+            valueListenable: textEditingController,
+            builder: (context, value, child) {
+              return textEditingController.text.isNotEmpty
```

Comment:
Why don't use `value` variable?
```suggestion
              return value.text.isNotEmpty
```